Tests/Data/PhaseField/beam3d_stag_1pcs.prj

```diff
@@ -0,0 +1,324 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<OpenGeoSysProject>
+    <mesh>beam3d.vtu</mesh>
+    <geometry>beam3d.gml</geometry>
+    <processes>
+        <process>
+            <name>PhaseField</name>
+            <type>PHASE_FIELD</type>
+            <coupling_scheme>staggered</coupling_scheme>
+            <integration_order>2</integration_order>
+            <constitutive_relation>
+                <type>LinearElasticIsotropic</type>
+                <youngs_modulus>E</youngs_modulus>
+                <poissons_ratio>nu</poissons_ratio>
+            </constitutive_relation>
+            <phasefield_parameters>
+                <residual_stiffness>k</residual_stiffness>
+                <crack_resistance>gc</crack_resistance>
+                <crack_length_scale>ls</crack_length_scale>
+                <kinetic_coefficient>M</kinetic_coefficient>
+                <history_field>H</history_field>
+            </phasefield_parameters>
+            <solid_density>rho_sr</solid_density>
+            <process_variables>
+                <phasefield>phasefield</phasefield>
+                <displacement>displacement</displacement>
+            </process_variables>
+            <specific_body_force>0 0 0</specific_body_force>
+        </process>
+    </processes>
+    <time_loop>
+        <global_process_coupling>
+            <max_iter> 1000 </max_iter>
+            <convergence_criteria>
+                <!-- convergence criterion for the first process -->
+                <convergence_criterion >
+                    <type>DeltaX</type>
+                    <norm_type>INFINITY_N</norm_type>
+                    <abstol>1.e-4</abstol>
+                    <reltol>1.e-10</reltol>
+                </convergence_criterion>
+                <!-- convergence criterion for the second process -->
+                <convergence_criterion >
+                    <type>DeltaX</type>
+                    <norm_type>INFINITY_N</norm_type>
+                    <abstol>1.e-4</abstol>
+                    <reltol>1.e-10</reltol>
+                </convergence_criterion>
+            </convergence_criteria>
+        </global_process_coupling>
+        <processes>
+            <process ref="PhaseField">
+                <nonlinear_solver>basic_newton_u</nonlinear_solver>
+                <convergence_criterion>
+                    <type>DeltaX</type>
+                    <norm_type>NORM2</norm_type>
+                    <reltol>1.e-4</reltol>
```

Comment:
Would a tighter tolerance be possible? If this is a simple test I'd think something around 1e-14 is easily achievable. The tighter the tolerances, here and in the ctest, the higher chance there is to spot a change in the process' behaviour.

Reply:
The tolerances have been made tighter (1e-14).